src/test/resources/features/11-allergy/allergies.feature

```diff
@@ -5,12 +5,29 @@ Feature: Allergies Management
     And a user clicks on Allergies link
     Then the system loads Allergies page
 
-  @selenium
+  @run
```

Comment:
please update to use selenium as in master, we nolonger use run

Reply:
Have addressed the recommendation